### PR TITLE
Chore/split org pages

### PR DESCRIPTION
**Pull request by joshenlim**
- Split organization settings pages into their own individual pages
- Deprecate use of PageContext for organization settings pages

### CODE REVIEW

**Review by saltcod — APPROVED** [review]

File: apps/www/lib/redirects.js [hunk 1]
```diff
@@ -1901,4 +1901,9 @@ module.exports = [
     source: '/legal/soc2',
     destination: 'https://forms.supabase.com/soc2',
   },
+  {
+    permanent: true,
+    source: '/:ref/settings',
```
Comment (saltcod) [inline apps/www/lib/redirects.js:1906]: Not certain if this is correct. Don't see a similar example in this file

**Review by saltcod — CHANGES_REQUESTED** [review]
Request changes — want to make sure we look at the redirect and the slug/settings link before we merge this.

**Comment by alaister** [comment]
Is there a problem with `getLayout` in the pages instead of just wrapping them?

File: apps/www/lib/redirects.js (hunk 1 above)
Comment (joshenlim) [reply apps/www/lib/redirects.js:1906]: hmm yeah i dont think this should be here - this file's for the marketing site

**Comment by joshenlim** [comment]
> Is there a problem with getLayout in the pages instead of just wrapping them?
Ahh i did it this way first cause i needed to pass in a `title` into the Layout component that's a variable from the UiStore, wasn't sure if that would work initially

**Comment by alaister** [comment]
@joshenlim, you should be able to make a new component (something like `OrgLayout`) that wraps `AccountLayout` and passes in the `title` and other required props from a `useStore` call. Then you can use your new layout inside `getLayout`.

I haven't tested it, but in my head it works 😅

**Comment by joshenlim** [comment]
Perfect that works :) pushed up the changes

**Review by saltcod — APPROVED** [review]

**Pull request merged**